app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -622,7 +605,6 @@
 //  _____________________________________________
 
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__s) {
-
     .product-info-price {
```

Comment:
seem static check failed at this point! In my point of view this code below totally find/valid

Reply:
> *:first-child totally valid syntax but core validate marked as failed/warning

Reply:
previously i have removed this code but i have changed my minds just in case it's can make regression bugs later. So i have reverted and add it back but afterwards magento code standards yells at this line as invalid code styles